### PR TITLE
2024 Execs

### PR DESCRIPTION
**Pull request by sebasptsch**
Add the 2024 Execs

### CODE REVIEW

File: docs/about/execs.md [hunk 1]
```diff
@@ -7,6 +7,18 @@ description: ProgSoc is run by a team of dedicated students who volunteer their
...
 
+## 2024 Executives
+
+* `President` Sebastian Pietschner
+* `Vice President` Oli Lenehan
+* `Treasurer` Jakob Tonnaer Lewis
+* `Secretary` Mitchell Kijurina
+* `Marketing` Vishwas Parpattegar
+* `Events`
+ * Audrey Santoso
```
Comment (sunsetkookaburra) [inline docs/about/execs.md:18]: Is mkdocs happy with 1 space for indents? Github refuses to indent anything less than four I think.

Comment (sebasptsch) [reply docs/about/execs.md:18]: Let me double check that.

Comment (sebasptsch) [reply docs/about/execs.md]: Fixed

**Review by sunsetkookaburra — APPROVED** [review]
All the names look right according to the minutes

**Pull request merged**